test/Passes/DefaultPipeline/default-tpp-passes.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: tpp-opt %s -default-tpp-passes -split-input-file | FileCheck %s
+// RUN: tpp-opt %s -default-tpp-passes -split-input-file 2>/dev/null | FileCheck %s
```

Comment:
Why is this necessary?

Reply:
The `linalg::detail::verifyContractionInterface` used in upstream generic matcher generates tons of diagnostics straight into stderr and makes output unusable.
I'll have to take a look at it upstream.

Reply:
This sounds like a bug. We don't want debug output unless explicitly requested. Diagnostics usually are enabled via some debug flag, maybe the macro is broken upstream?

Reply:
It just straight up dumps info through `op->emitError`. It's a questionable design I think but the call in question could be replaced with `isContractionInterfaceImpl` to avoid the issue.

Reply:
That's not right, we can fix it upstream.

Reply:
Opened PR upstream with a fix.

Reply:
I'll land this for now and try to remember to clean it up after the next LLVM bump.